libs/state/docs/testing.md

```diff
@@ -0,0 +1,235 @@
+# Testing
+
+This document provides a brief overview on how to set up unit tests with `jest` when using the `@rx-angular/state`
+package.
+
+> Make sure you are familiar with the basics of how to test angular applications.
+> You can read more about it in the [official testing docs](https://angular.io/guide/testing).
+
+## RxState
+
+There are two ways you can test `RxState`. Depending on your use case, you maybe want
+to test the behavior of a whole `component`, or test the `state` and it's transformations on
+its own.
+
```

Comment:
I would add something like: 

Note that you want your tests to avoid including implementation details, keep in mind that testing a component with DOM nodes instead of component instance is considered as a best practice.

Reply:
added something, check again :)